src/pyramid_sanity/__init__.py

```diff
@@ -1,44 +1,19 @@
-"""Main integration into Pyramids automatic configuration hoovering."""
```

Comment:
I remove the docstring as they're nothing left in the file other than an `includeme()` and what an `includeme()` is / does is already defined by Pyramid